packages/math-input/src/components/compute-layout-parameters.ts

```diff
@@ -19,7 +19,7 @@
  * might need to be.
  */
 
-import {DeviceTypes, DeviceOrientations, LayoutModes} from "../consts";
+import {DeviceType, DeviceOrientation, LayoutMode} from "../enums";
```

Comment:
Once I got to the end of these changes, `consts.ts` only contained enums, so this seemed like a logical filename change. :)